bgpd/bgp_vty.c

```diff
@@ -14466,8 +14467,8 @@ DEFUN (show_community_list_arg,
 	if (argv_find(argv, argc, "ip", &idx)) {
 		vty_out(vty, "This config option is deprecated, and is scheduled for removal.\n");
 		vty_out(vty, "if you are using this please migrate to the below command.\n");
-		vty_out(vty, "'show bgp community-list <(1-500)|WORD>'\n");
-		zlog_warn("Deprecated option: 'ip show community-list <(1-500)|WORD>' being used");
+		vty_out(vty, "'show bgp community-list <(1-500)|WORD> detail'\n");
+		zlog_warn("Deprecated option: 'show ip community-list <(1-500)|WORD>' being used");
```

Comment:
shouldn't be `Deprecated option: 'show ip community-list <(1-500)|WORD> detail' being used`? (with detail keyword)

Reply:
@ton31337 : Thanks for your review.
Actually it's a warning message for logging purpose only. Also show ip community-list <(1-500)|WORD> is a valid command only, so i feel detail is not required.
The detail keyword is required only for bgp, like if we have show bgp communitity-list. 
Please check it once.